broadcast/internal/arc/arc_submit_tx.go

```diff
@@ -203,3 +227,42 @@ func validateSubmitTxResponse(model *broadcast.SubmittedTx) error {
 
 	return nil
 }
+
+func convertToEfTransaction(tx *SubmitTxRequest) error {
+	junglebusClient, err := junglebus.New(
+		junglebus.WithHTTP("https://junglebus.gorillapool.io"),
+	)
+	if err != nil {
+		return err
+	}
+	transaction, err := bt.NewTxFromString(tx.RawTx)
+	if err != nil {
+		return err
+	}
+	for _, input := range transaction.Inputs {
+		txid := input.PreviousTxIDStr()
```

Comment:
Just a suggestion - maybe consider extract loop "body" as a separated method with descriptive name?
I mean lines 243-254.

Reply:
Okay, good point, please check if what I did is what you meant :)